apps/contrib/templates/a4_candy_contrib/includes/form_checkbox_field.html

```diff
@@ -3,7 +3,7 @@
 <div class="form-check">
     <label class="form-check__label">
         {{ field }}
-        {{ field.label }}
+        {{ field.label }}{% if field.field.required %}<span role="presentation" title="{% trans 'This field is required' %}">*</span>{% endif %}
```

Comment:
no clue but only place I know we have a required checkbox, (sign up form) we don't use it?!

Reply:
I think it's from mB so we could check there?

Reply:
@fuzzylogic2000 Was it maybe, because for the image_right_of_use, it is only required if there is an actual image (and that is checked in the clean)? So maybe for organisations_terms_of_use the form field could be required?

Reply:
Haha, ok, maybe should have read this conversation til the end, sorryyy :see_no_evil: